### ISSUE TITLE
New Relic sourcemap upload error

### ISSUE THREAD

**Issue by marr** [1stdibs/new-relic-source-map-webpack-plugin#14]
I have tried using `source-map` and `hidden-source-map` for my devtool options, and they both emit: 
>No sourcemaps were found. Check if sourcemaps are enabled: https://webpack.js.org/configuration/devtool/

during build. I am using the version in `master` right now (db4a293d0f210503d4fcd0e42a17104add98f477).

**Comment by marr** [comment]
Is there a specific version of webpack this needs to be used with? After much digging, I see the line in _uploadSourceMap.js_ always true when checking `!fileObj.children`, therefore never actually trying to upload the file.
```
    const fileObj = assets[item];
    if (!fileObj.children) {
        return Promise.resolve();
    }
```

**Comment by niqqq25** [comment]
Hey, can you provide your minimal webpack configs, which will be just enough to reproduce the following issue.

**Comment by kostasmanionis** [comment]
Might be fixed with `v1.2.0`. Closing.
